mmv1/third_party/terraform/website/docs/r/container_node_pool.html.markdown

```diff
@@ -262,6 +262,10 @@ cluster.
   Specifying COMPACT placement policy type places node pool's nodes in a closer
   physical proximity in order to reduce network latency between nodes.
 
+* `policy_name` - (Optional) If set, refers to the name of a custom resource policy supplied by the user.
```

Comment:
This should also be documented in the container_cluster markdown I believe? Though I'm not seeing the whole placement_policy object in there, maybe it was missed when it was added

Reply:
The `placement_policy` is a node pool setting - it can be passed during cluster creation as documented [here](https://cloud.google.com/kubernetes-engine/docs/how-to/compact-placement#create_a_compact_placement_policy_in_a_new_cluster), but it only impacts the default created node pool. Should it be included in container_cluster markdown then?